redash/query_runner/elasticsearch.py

```diff
@@ -43,6 +45,13 @@
 }
 
 
+def get_index_from_sql(sql):
+    matches = re.search(r"from ([\w\d]+)", sql, flags=re.IGNORECASE)
```

Comment:
It does not match an index of the type `metadata_0_2018-08-29_18:40:21`, I would add `([\w\d-:]+)`